packages/extension/src/view/devtools/components/tabContent/cookies/cookiesListing/index.tsx

```diff
@@ -29,25 +29,44 @@ import { useFilterManagementStore } from '../../../../stateProviders/filterManag
 import ChipsBar from '../cookieFilter/chips';
 import CookieTopBar from '../cookieTopBar';
 import FiltersList from '../cookieFilter';
+import { type CookieTableData } from '../../../../cookies.types';
 
 const CookiesListing = () => {
-  const { selectedFrame } = useCookieStore(({ state }) => ({
+  const { cookies, selectedFrame, tabFrames } = useCookieStore(({ state }) => ({
+    cookies: state.tabCookies,
     selectedFrame: state.selectedFrame,
+    tabFrames: state.tabFrames,
   }));
 
   const filteredCookies = useFilterManagementStore(
     ({ state }) => state.filteredCookies
   );
 
+  const frameFilteredCookies = useMemo(() => {
```

Comment:
This is already calculated in frame management store. Calculation of `cookiesAvailable` can also be moved to that store.

Reply:
Fixed in this [99cf5a2](https://github.com/GoogleChromeLabs/ps-analysis-tool/pull/108/commits/99cf5a28ae719eb15c843ce6b4a39977f2cb1794)